### ISSUE TITLE
possible bug in interactive v3 docs related to bill search parameters 

### ISSUE THREAD

**Issue by akkana** [openstates/issues#378]
This is the right template to select if you're reporting an issue with the functionality of the v3 API.
If this is a data issue, please select data issue instead!

**API Query (be sure to omit API key!!!):**
```
https://v3.openstates.org/bills?jurisdiction=ocd-jurisdiction%2Fcountry%3Aus%2Fstate%3Anm%2Fgovernment&session=2021&sort=updated_desc&page=1&per_page=20&apikey=...
```

**Please describe the issue:**
Please consider adding a way to search for bills by identifier, e.g. "SB 304" with a given jurisdiction and session.
It looks like the only way is to iterate through pages of 20 bills at a time to get all bills for the given jurisdiction and session, in order to find the internal id so I can query for the bill's details. Full text search doesn't help: it doesn't seem to match what's in the identifier field and it ignores jurisdiction and session. Is there a way that I'm not seeing to find details for NM 2021 SB304?

By the way, your interactive API documentation is wonderful. I've never seen API docs like that before.

**Comment by jamesturk** [comment]
Thanks for reporting this, the search should already work this way, can you give an example of where

``jurisdiction=x&session=y&q=z ``

doesn't work?  I'll investigate from there.

**Comment by jamesturk** [comment]
Oh that's certainly interesting, I'll rename this ticket and investigate that a bit later.

**Comment by akkana** [comment]
There's a similar problem with /people. If I expand the /people search area, enter a jurisdiction of ocd-jurisdiction/country:us/state:nm/government plus my api key, when I click Execute the query correctly shows nm:
`https://v3.openstates.org/people?jurisdiction=ocd-jurisdiction%2Fcountry%3Aus%2Fstate%3Anm%2Fgovernment&org_classification=legislature&page=1&per_page=10&apikey=...`
but the single result returned has jurisdiction id ocd-jurisdiction/country:us/state:nc/government. It also says "validation error" after the json, though above the json it says 200 successful response.

If I run the suggested curl command, though, it returns nothing:
`{"results":[],"pagination":{"per_page":10,"page":1,"max_page":1,"total_items":0}`
Maybe the database doesn't have people for NM yet?
(If not, I could help with that: I run my own scraper, nmbilltracker.com, and the reason I'm looking at these queries is to see whether I could merge the two projects, so I could put my development effort into openstates instead. I do have working scraping code for NM legislators and committees as well as bills.)

**Comment by jamesturk** [comment]
Can you remove org_classification= from the query?

And maybe provide some screenshots/etc. of the query builder?  I'm not sure where you're getting these results, but probably something weird about the UI of the query builder if I had to guess.

We do have NM data (https://openstates.org/nm) - and it is accessible via the API, so we can get this sorted out.

**Comment by akkana** [comment]
Sure enough, removing the org_classification fixes it in both cases and I get NM results both in curl and on the docs site. "upper" and "lower" work too, just not "legislature" (which I had expected would mean lower+upper).

I fiddled around with parameters: I think the original problem was a result of chamber=S classification=B -- I guess I needed classification=bill and none of them seems to have chamber set, so I needed to leave that blank. I thought that I saw the nc results even before I filled in chamber and classification, but I can't reproduce that, so maybe not.
